packages/amplication-data-service-generator/src/admin/static/src/user/RoleSelect.tsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import { SelectField, SelectFieldProps } from "@amplication/design-system";
+//@ts-ignore
+import { EnumRoles } from "./EnumRoles";
+
+const OPTIONS = Object.keys(EnumRoles).map((key) => ({
```

Comment:
Should use `Object.entries()` and use keys as labels and values as values.

Reply:
But I think it would be easier to just create an options array using the role display name

Reply:
We should be able to fetch the roles display names somehow. In the admin the developer can set both a name and a display name for each role

Reply:
> But I think it would be easier to just create an options array using the role display name

Please do so